GoogleForCreators/web-stories-wp
# Templates: improve text element usage

## Context
Improves templates by using Fill / Highlight / Border features of the text element instead of using separate shape for the same effect.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
Buttons etc. in templates that previously used a text element + background shape for the background / border now use Fill / border features instead. Note that this was not possible in 100% places, e.g. in some places it seemed like the text was purposefully not vertically centred, making it not possible to use the Fill background feature. 


## Testing Instructions






This PR can be tested by following these steps:

1. Check some templates (see the Files changed to see which templates have been changed) and see that where possible, button-like elements and other texts with background now consist of 1 text element with Fill background / border instead of text + shape.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #12032
